assets/js/modules/tagmanager/setup.js

```diff
@@ -297,72 +350,80 @@ class TagmanagerSetup extends Component {
 
 			global.googlesitekit.modules.tagmanager.settings = savedSettings;
```

Comment:
This still makes me sad (I know it was here before) - can't wait for having a datastore.